### PR TITLE
Final batch of pylab example moves

### PR DESCRIPTION
**Pull request by dstansby**
Again, first commit is moves, second commit is PEP8, third commit is link fixing.

### CODE REVIEW

File: examples/misc/anchored_artists.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 
 
 class AnchoredText(AnchoredOffsetbox):
```
Comment (QuLogic) [inline examples/misc/anchored_artists.py]: I wonder if we need this `class` when there's a `matplotlib.offsetbox.AnchoredText` implementation already?

File: examples/misc/anchored_artists.py [hunk 2]
```diff
@@ -26,8 +27,8 @@ class AnchoredSizeBar(AnchoredOffsetbox):
     def __init__(self, transform, size, label, loc,
```
Comment (QuLogic) [inline examples/misc/anchored_artists.py]: And `AnchoredSizeBar` is in `mpl_toolkits.axes_grid1.AnchoredSizeBar`.

File: examples/images_contours_and_fields/demo_bboximage.py [hunk 3]
```diff
@@ -37,7 +37,7 @@
         not m.startswith(('spectral', 'Vega'))  # Skip deprecated colormaps.
     )
 
-    #fig.subplots_adjust(top=0.99, bottom=0.01, left=0.2, right=0.99)
+    # fig.subplots_adjust(top=0.99, bottom=0.01, left=0.2, right=0.99)
```
Comment (NelleV) [inline examples/images_contours_and_fields/demo_bboximage.py]: I guess at some point we should clean this up and just delete those lines of codes.

**Comment by LindyBalboa** [comment]
Partially resolves #8922

**Pull request merged**